eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Mgmt/src/Providers/ResourceClientProvider.cs

```diff
@@ -59,9 +59,9 @@ public ResourceClientProvider(InputClient inputClient)
 
             _contextualParameters = GetContextualParameters(requestPath);
```

Comment:
remove this field since a new property is added, and assign the property directly.